core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -25,54 +25,39 @@
 public class OpenSearchFunctions {
   public void register(BuiltinFunctionRepository repository) {
     repository.register(match());
+    repository.register(match_phrase());
   }
 
   private static FunctionResolver match() {
     FunctionName funcName = BuiltinFunctionName.MATCH.getName();
+    // At most field, query, and all optional parameters
+    final int matchMaxNumParameters = 14;
```

Comment:
if you move these into a public static final variable, it can be imported into other files. 
At least the least, we could then write tests to test these boundary cases using an imported value (like error scenarios where the max number of parameters is exceeeded).

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/48/commits/e6f7e2fc36f13e9904dab9ef106c49eec89ef25d